### ISSUE TITLE
delete button deletes non selected note

### ISSUE THREAD

**Issue by eddex** [notable/notable#633]
- **OS: Windows 10**:
- **Notable Version: 1.4.0**:

### Current behavior

Unfortunately I'm not able to reproduce the issue. What I did was:

1. select a notebook `Notebooks/work`
2. create select an existing note called `Notes`
3. create a new note
4. add a tag to the new note called `Notebooks/Notebooks`

Then I'm not entirely sure anymore what L did, I think I selected the new tag `Notebooks/Notebooks` and then deleted it. The selection went back to the previously selected `Notebooks/work` tag and the new `untitled` note was still open and in edit mode.

5. press the delete button
6. instead of the currently visible `untitled` note, the `Notes` note was deleted.
7. after pressing the delete button again, the `untitled` note was deleted as well
8. in the `Trash` were 2 notes. Both called `untitled` with the content of the newly created, untitled note. (it is possible that one of them was there before..)

might be related to #603

### Expected behavior

the selected note gets deleted

**Comment by fabiospampinato** [comment]
I can't reproduce with the provided steps. @eddex is there any more detail that you can give me? Like perhaps you performed those steps very quickly or something? Did it ever happen again?

**Comment by eddex** [comment]
This just happened once and I wasn't able to reproduce it either. For now I added a git repo to my .notable folder to prevent losing notes in the future. I'll keep an eye out and if it happens again, I'll add further information here.

Are there any log files I could have a look at or attach here?

**Comment by fabiospampinato** [comment]
> Are there any log files I could have a look at or attach here?

Nope, unfortunately. But it looks like having some sort of logs of the detected FS events might be helpful 🤔

**Comment by fabiospampinato** [comment]
@eddex did it ever happen again?

**Comment by eddex** [comment]
Nope

**Comment by fabiospampinato** [comment]
I'll be glad to take a closer look at this if anybody can reproduce and/or can provide more reliably reproduction steps.
